docs/ManualMocks.md

```diff
@@ -60,6 +60,9 @@ called. To opt out of this behavior you will need to explicitly call
 `jest.unmock('moduleName')` in tests that should use the actual module
 implementation.
 
+> Warning: To make mock works properly we have to put the call
```

Comment:
Can we change this to `Note` and fix the typo for `works?

Reply:
> Note: in order to mock properly, Jest needs `jest.mock('moduleName')` to be in the same scope as the `require/import` statement.

Reply:
Yep, give some minutes